build/rpms/kubelet.spec

```diff
@@ -0,0 +1,31 @@
+Name: kubelet
+Version: OVERRIDE_THIS
+Release: 00
+License: ASL
+Summary: Container Cluster Manager
+
+URL: https://kubernetes.io
+
+Requires: iptables >= 1.4.21
+Requires: kubernetes-cni >= 0.5.1
+Requires: socat
+Requires: util-linux
+Requires: ethtool
+Requires: iproute
+Requires: ebtables
+
+%description
+The node agent of Kubernetes, the container cluster manager.
+
+%install
+
+install -m 755 -d %{buildroot}%{_bindir}
+install -m 755 -d %{buildroot}%{_sysconfdir}/systemd/system/
+install -m 755 -d %{buildroot}%{_sysconfdir}/kubernetes/manifests/
+install -p -m 755 -t %{buildroot}%{_bindir} kubelet
+install -p -m 755 -t %{buildroot}%{_sysconfdir}/systemd/system/ kubelet.service
+
+%files -n kubelet
```

Comment:
That is true. At a SPEC file, the `-n` flag at the `%files` directive is used  in conjunction with the `%package `directive (when it uses the `-n` flag as well), in order for the resulting rpm to have a different name than the one specified with the `Name` tag. Another use for it, is creating subpackages with varying names from only one [SPEC](https://fedoraproject.org/wiki/Packaging:Python#Example_common_spec_file).

All in all the `-n kubelet` can be omitted here.